apps/example-app-karma/src/test.ts

```diff
@@ -11,7 +11,9 @@ beforeAll(() => {
 declare const require: any;
 
 // First, initialize the Angular testing environment.
-getTestBed().initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting());
+getTestBed().initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting(), {
+  teardown: { destroyAfterEach: false },
```

Comment:
automatically set to false. We might want to set it to true?

Reply:
Yes, you can remove this config - I would like to use the default settings.